integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,246 @@
+# Integrating Kobiton in app development pipeline with Travis CI
+This document will guide you to prepare an app automation test project on GitHub and use Travis CI to automatically fetch the latest version and execute tests on Kobiton devices everytime you push a commit to GitHub.
+
+## Table of contents 
+[Prerequisites](#prerequisites)
+
+[Preparation](#preparation)
+
+[1. Configuring Travis CI with GitHub repository](#1-configuring-travis-ci-with-github-repository)
+
+[2. Configuring automation test script](#2-configuring-automation-test-script)
+
+[3. Adding required enviroment variables to Travis CI](#3-adding-required-enviroment-variables-to-travis-ci)
+
+[4. Configuring to test latest app version on Kobiton devices](#4-configuring-to-test-latest-app-version-on-kobiton-devices)
+  - [4.1. Fetching the latest app version and app download URL using Hockey App API](#41-fetching-the-latest-app-version-and-app-download-url-using-hockey-app-api)
+  - [4.2 Adding test script execution command](#42-adding-test-script-execution-command)
+  - [4.3 Setting Desired Capabilities](#43-setting-desired-capabilities)
+  
+[5. Executing your build](#5-executing-your-build)
+
+[6. Fetching session data through REST API](#6-fetching-session-data-through-rest-api)
+
+[7. Feedback](#7-feedback)
+
+## Prerequisites
+  - Kobiton account
+  
+  Please visit https://portal.kobiton.com/register to create new account.
+
+## Preparation
+We are going to assume you've already provided a full configured deployment pipeline from your React Native project to Hockey App using Travis CI.
```

Comment:
I think you should mention that the application must have been created on HockeyApp (even placeholder)